chapters/ch19.md

```diff
@@ -4,23 +4,23 @@
 ->
-* Khi bạn bắt đầu một dự án mới, đặc biệt nếu đó là một lĩnh vực mà bạn không phải là chuyên gia, nó rất khó để dự đoán được những hướng giải quyết triển vọng nhất.
+* Khi bạn bắt đầu một dự án mới, đặc biệt nếu đó là một lĩnh vực mà bạn không phải là chuyên gia, sẽ rất khó để dự đoán những hướng giải quyết triển vọng nhất.
 
 * So don’t start off trying to design and build the perfect system. Instead build and train a basic system as quickly as possible—perhaps in a few days. Then use error analysis to help you identify the most promising directions and iteratively improve your algorithm from there.
 ->
-* Vì vậy đừng bắt đầu với việc thiết kế và xậy dựng một hệ thống hoàn hảo. Thay vào đó hãy xây dựng và huấn luyện một hệ thống cơ bản một cách nhanh nhất có thể-thậm chí chỉ trong một vài ngày. Sau đó, sử dụng phân tích lỗi để giúp bạn xác định những hướng triển vọng và lặp đi lặp lại cải thiện thuật toán của bạn từ đó.
+* Vì vậy đừng cố bắt đầu với việc thiết kế và xậy dựng một hệ thống hoàn hảo. Thay vào đó hãy xây dựng và huấn luyện một hệ thống cơ bản một cách nhanh nhất có thể-thậm chí chỉ trong một vài ngày. Sau đó, sử dụng phân tích lỗi để giúp bạn xác định những hướng đi triển vọng và lặp đi lặp lại việc cải thiện thuật toán của bạn từ đó.
 
 * Carry out error analysis by manually examining ~100 dev set examples the algorithm misclassifies and counting the major categories of errors. Use this information to prioritize what types of errors to work on fixing.
 ->
-* Thực hiện phân tích lỗi bằng cách kiểm tra thủ công khoảng 100 mẫu trong tập phát triển mà thuật toán phân loại sai và đếm những loại lỗi chính. Sử dụng thông tin này để ưu tiên các loại lỗi để khắc phục.
+* Thực hiện phân tích lỗi bằng cách kiểm tra thủ công khoảng 100 mẫu trong tập phát triển mà thuật toán phân loại sai và điểm qua những hạng mục lỗi chính. Sử dụng thông tin này để sắp xếp thứ tự ưu tiên khắc phục các loại lỗi.
 
 * Consider splitting the dev set into an Eyeball dev set, which you will manually examine, and a Blackbox dev set, which you will not manually examine. If performance on the Eyeball dev set is much better than the Blackbox dev set, you have overfit the Eyeball dev set and should consider acquiring more data for it.
 ->
-* Xem xét việc tách nhỏ tập phát triển thành một tập phát triển Eyeball, bạn sẽ kiểm tra thủ công, và một tập phát triển Blackblox , cái mà bạn sẽ không kiểm tra thủ công. Nếu hiệu quả trên tập phát triển Eyeball tốt hơn rất nhiều so với tập phát triển Blackbox, bạn đã overfit tập phát triển Eyeball và nên xem xét việc thu thập thêm dữ liệu cho nó.
+* Xem xét việc tách tập phát triển thành một tập phát triển Eyeball cho việc kiểm tra thủ công, và một tập phát triển Blackblox  mà bạn sẽ không kiểm tra thủ công. Nếu hiệu quả trên tập phát triển Eyeball tốt hơn rất nhiều so với tập phát triển Blackbox, bạn đã overfit tập phát triển Eyeball và nên xem xét việc thu thập thêm dữ liệu cho tập này.
```

Comment:
"Blackbox  mà" -->"Blackbox mà" (thừa space). 

"so với tập phát triển" --> "so với trên tập phát triển"

Reply:
- performance dịch là chất lượng thì chuẩn hơn alf "hiệu quả"

Reply:
> * performance dịch là chất lượng thì chuẩn hơn alf "hiệu quả"

Yeah, cái này có thể chỉnh lại toàn bộ trong giai đoạn hai. Từ đầu mình đã dùng "hiệu quả" rồi.